### ISSUE TITLE
Red Eye Reduction not working in Ubuntu 20.04

### ISSUE THREAD

**Issue by hankschwie** [Aries85/LightZone#259]
Hi!

Trying to use the Red Eye Reduction Tool always ends in an error, no matter what file format I use:

```
failed tile 1:1
java.lang.UnsupportedOperationException: Unsupported data type: 3
	at com.lightcrafts.jai.opimage.RedMaskOpImage.computeRect(RedMaskOpImage.java:60)
	at javax.media.jai.PointOpImage.computeTile(PointOpImage.java:914)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at javax.media.jai.PlanarImage.cobbleByte(PlanarImage.java:2918)
	at javax.media.jai.PlanarImage.getData(PlanarImage.java:2169)
	at javax.media.jai.AreaOpImage.computeTile(AreaOpImage.java:486)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at javax.media.jai.PlanarImage.cobbleByte(PlanarImage.java:2918)
	at javax.media.jai.PlanarImage.getData(PlanarImage.java:2169)
	at javax.media.jai.PlanarImage.getExtendedData(PlanarImage.java:2508)
	at javax.media.jai.AreaOpImage.computeTile(AreaOpImage.java:485)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at javax.media.jai.RenderedOp.getTile(RenderedOp.java:2258)
	at javax.media.jai.PointOpImage.computeTile(PointOpImage.java:911)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at javax.media.jai.PointOpImage.computeTile(PointOpImage.java:911)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at com.lightcrafts.jai.opimage.BlendOpImage.getTile(BlendOpImage.java:207)
	at javax.media.jai.RenderedOp.getTile(RenderedOp.java:2258)
	at com.lightcrafts.model.ImageEditor.BlendedOperation$BlendedTransform$CachedImage.getTile(BlendedOperation.java:318)
	at com.sun.media.jai.opimage.CropOpImage.getTile(CropOpImage.java:118)
	at javax.media.jai.PointOpImage.computeTile(PointOpImage.java:911)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at javax.media.jai.PointOpImage.computeTile(PointOpImage.java:911)
	at com.sun.media.jai.util.SunTileScheduler.scheduleTile(SunTileScheduler.java:899)
	at javax.media.jai.OpImage.getTile(OpImage.java:1125)
	at com.sun.media.jai.util.RequestJob.compute(SunTileScheduler.java:242)
	at com.sun.media.jai.util.WorkerThread.run(SunTileScheduler.java:463)
```
I'm runnig the current version:

```
LightZone version 4.2.3 ...
with options :
openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-Ubuntu-0ubuntu2.20.04)
OpenJDK 64-Bit Server VM (build 11.0.11+9-Ubuntu-0ubuntu2.20.04, mixed mode, sharing)
```
Any ideas what went wrong?

btw: Is this the current issue tracker? I couldn't find any issues at [ktgw0316's LightZone page ](https://github.com/ktgw0316/LightZone).

so long
hank

**Comment by ktgw0316** [comment]
@hankschwie Thank you for the report.  I don't know why the Red Eye tool had worked fine in older versions of LZ.  Anyway, I modified the code to fix this issue.

**Comment by hankschwie** [comment]
Hi!
Thanks for the speedy response and the fix! I tried 4.2.0 as well and i didn't work there either; checked with Java 17 , too - same result.
